test/e2e_source_map_test.ts

```diff
@@ -229,21 +255,32 @@ function tsickleCompiler(
 }
 
 function compile(
-    sources: Map<string, string>, outFile = 'output.js',
-    filesNotToProcess = new Set<string>()): {compiledJS: string, sourceMap: SourceMapConsumer} {
+    sources: Map<string, string>, outFile = 'output.js', filesNotToProcess = new Set<string>(),
+    inlineSourceMap = false): {compiledJS: string, sourceMap: SourceMapConsumer} {
   const resolvedSources = new Map<string, string>();
   for (const fileName of toArray(sources.keys())) {
     resolvedSources.set(ts.sys.resolvePath(fileName), sources.get(fileName));
   }
 
   const diagnostics: ts.Diagnostic[] = [];
 
+  let compilerOptions: ts.CompilerOptions;
+  if (inlineSourceMap) {
+    compilerOptions = {
+      inlineSourceMap: inlineSourceMap,
+      outFile: outFile,
+      experimentalDecorators: true
+    };
+  } else {
+    compilerOptions = {sourceMap: true, outFile: outFile, experimentalDecorators: true};
```

Comment:
if you write this with an extra comma:
```
experimentalDecorators: true,};
```
clang-format will lay it out linearly which makes it match the line above.